### DISCUSSION TITLE
LDAP: Get directus Role from LDAP attributes

### DISCUSSION THREAD

**Discussion by LasseRosenow** [directus/directus#12730] [Archive]
Hello, we have an LDAP instance and its user have many many custom attributes that we use in other products to specify their permissions.

From what I can see directus is usually using `active directory groups` to apply `directus roles` to LDAP users.
But we need to map the ldap attributes so that we can have a directus role.

For example: we have these attributes:
staffCategory: "secretary"
departmentName: "management"

We would like to specify something like this:
role = `staffCategory + "-" + departmentName`
in LDAP.

Now the role this user would have would be
secretary-management

This seems to be quite normal in other products but I have no idea how to do this in directus, so is there any way to do it?

**Comment by aidenfoxx** [comment]
There is currently no way to do this. You can only query user groups by one attribute at the moment.
